README.md

```diff
@@ -1,6 +1,6 @@
 # Prometheus Operator
-[![Build Status](https://travis-ci.org/coreos/prometheus-operator.svg?branch=master)](https://travis-ci.org/coreos/prometheus-operator)
```

Comment:
I would really like to mention that this used to be under coreos org before, as to not confuse people. Something like previously known as coreos/prometheus-operator. WDYT?

Reply:
We need a complete overhaul of docs not only `README.md`, but also `CONTIBUTING.md` etc. This PR is only to get basic working solution.
I might take a stab at it after all migrations are finished.

Reply:
Understood I meant more in between the overhaul and this PR, just so folks are aware who might be coming from any announcements or noticed that things changed.

Reply:
Added a small note.